MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionPresenter.h

```diff
@@ -330,6 +337,12 @@ protected slots:
   // name of the workspace with the vanadium integration (of spectra)
   static const std::string g_vanIntegrationWSName;
 
+  // vector holding directory of focused bank file
+  // std::vector<std::string> static m_fitting_runno_dir_vec;
+
+  // indentifier for fitting multi-run or single run input
+  // bool static m_fittingMutliRunMode;
+
```

Comment:
If these commented out variables are not going to be used, let's just remove them.